Packages/com.unity.inputsystem/InputSystem/Devices/InputDevice.cs

```diff
@@ -247,14 +247,27 @@ public bool canRunInBackground
                 // In the editor, "background" refers to "game view not focused", not to the editor not being active.
                 // So, we modulate canRunInBackground depending on how input should behave WRT game view according
                 // to the input settings.
-                #if UNITY_EDITOR
+#if UNITY_EDITOR
```

Comment:
Personally I find this multi function spanning #if a bit hard to parse. Would love it if this could be simplified.

Reply:
Agree. From a readability perspective, it's not great. Can this be improved @bmalrat ?